homestar-runtime/src/event_handler.rs

```diff
@@ -213,3 +243,13 @@ where
         }
     }
 }
+
+/// Poll cache for expired entries
+async fn poll_cache(cache: Arc<Cache<String, CacheValue>>) {
+    let mut interval = tokio::time::interval(Duration::from_secs(1));
```

Comment:
let's make this configurable, maybe under monitoring?

Reply:
Done. Put it in the network config for now, but we can shuffle it around later.